tsconfig.json

```diff
@@ -2,12 +2,12 @@
   "compilerOptions": {
     "target": "ES2020",
     "useDefineForClassFields": true,
-    "lib": ["ES2020", "DOM", "DOM.Iterable"],
+    "lib": ["ES2023", "DOM", "DOM.Iterable"],
```

Comment:
is there a need to change ES to another version?

Reply:
yes, i guess
 it was recommended by one of our mentors